### PR TITLE
MUL-169 - adds validation mapping for original price and minimum price

### PR DESCRIPTION
**Pull request by MacBalc**

### CODE REVIEW

File: src/Validator/ProductListingPriceValidator.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+use BitBag\OpenMarketplace\Entity\ProductListing\ProductListingPriceInterface;
+use BitBag\OpenMarketplace\Validator\Constraint\ProductListingPriceConstraint;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+final class ProductListingPriceValidator extends ConstraintValidator
+{
+    public function __construct(private ChannelRepositoryInterface $channelRepository)
```
Comment (Wojdylak) [inline src/Validator/ProductListingPriceValidator.php:23]: I think we should use older method, for code-style consistency

File: src/Validator/ProductListingPriceValidator.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace BitBag\OpenMarketplace\Validator;
+
+use BitBag\OpenMarketplace\Entity\ProductListing\ProductListingPriceInterface;
+use BitBag\OpenMarketplace\Validator\Constraint\ProductListingPriceConstraint;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+final class ProductListingPriceValidator extends ConstraintValidator
```
Comment (Wojdylak) [inline src/Validator/ProductListingPriceValidator.php:21]: No tests

File: src/Validator/ProductListingPriceValidator.php (hunk 1 above)
Comment (MacBalc) [reply src/Validator/ProductListingPriceValidator.php]: property promotion przerobione na tradycyjny mechanizm inicjowania

File: src/Validator/ProductListingPriceValidator.php (hunk 2 above)
Comment (MacBalc) [reply src/Validator/ProductListingPriceValidator.php:21]: test dodany

**Review by mpysiak — APPROVED** [review]

**Pull request merged**